streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -916,6 +923,12 @@ private void handleRemovedTasksFromStateUpdater() {
                 stateUpdater.add(task);
             } else if (tasks.removePendingTaskToCloseClean(task.id())) {
                 closeTaskClean(task, tasksToCloseDirty, taskExceptions);
+            } else if ((inputPartitions = tasks.removePendingTaskToCloseReviveAndUpdateInputPartitions(task.id())) != null) {
+                if (closeTaskClean(task, tasksToCloseDirty, taskExceptions)) {
+                    task.revive();
+                    task.updateInputPartitions(inputPartitions, topologyMetadata.nodeToSourceTopics(task.id()));
+                    stateUpdater.add(task);
```

Comment:
Where is the task initialized?
The task is closed cleanly and the call to `revive()` sets the task to `CREATED`. However, the task is never initialized after that.